usage.md

```diff
@@ -0,0 +1,87 @@
+# Usage as an instance control
```

Comment:
`Usage as an information model instantiation according to companion specifications `

Reply:
Updated